### PR TITLE
upgrade to flow 0.72.0

### PR DESCRIPTION
**Pull request by atabel**

### CODE REVIEW

File: src/PluginsRegistry.js [hunk 1]
```diff
@@ -69,7 +69,7 @@ export default class PluginsRegistry {
   /**
    * Call `onUpdate` hooks.
    */
-  onUpdate(data: Object, rule: Rule, sheet: StyleSheet): void {
+  onUpdate(data: ?Object, rule: ?Rule, sheet: StyleSheet): void {
```
Comment (kof) [inline src/PluginsRegistry.js]: where is it comming from? I don't think those can be optional

Comment (TrySound) [reply src/PluginsRegistry.js]: Seems like data argument can be `undefined`. I added checks and integration tests are failed when data is always object.

Comment (atabel) [reply src/PluginsRegistry.js]: `data` is optional here https://github.com/cssinjs/jss/blob/master/src/RuleList.js#L137

**Comment by kof** [comment]
closing this one as we just merged this https://github.com/cssinjs/jss/pull/716

**Pull request closed without merging**